test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -605,7 +605,8 @@ protected Set<String> preserveILMPolicyIds() {
             ".fleet-file-data-ilm-policy",
             ".fleet-files-ilm-policy",
             ".deprecation-indexing-ilm-policy",
-            ".monitoring-8-ilm-policy"
+            ".monitoring-8-ilm-policy",
```

Comment:
ℹ️ All "default" policies are listed here so they are not cleansed when the test is finished. Adding ours to avoid failure.